components/GameCard/GameCard.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+import { BsPersonCircle } from "react-icons/bs";
+
+export default function GameCard({ type, name, date, time, place, court }) {
+  return (
+    <section>
+      <StyledGameHeadline>{type}</StyledGameHeadline>
+      <StyledCardContainer>
+        <StyledCardPlayer>
+          <p>
+            <BsPersonCircle /> {name}
+          </p>
+        </StyledCardPlayer>
+        <StyledCardContent>
```

Comment:
It would be better semantically to use a list here instead of nested articles:
```
<StyledCardContent>
   <article>...
   <article>...
</StyledCardContent>
```
You can also use `list-style: none` to achieve a similar styling.